app/src/main/res/values/strings_whatsnew.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="whatsnew_activity_title">What\'s New in CometOTP</string>
```

Comment:
Copy paste error? ;)

Reply:
Lol! 😂 😂 😂. Was testing it with a fresh fork!